charts/spilo/values.yaml

```diff
@@ -48,9 +48,18 @@ continuousArchive:
     storageClass: netapp-file-backup
     size: 500Mi
 
+  # If choose s3 as the storage type, the secret has to be created first. 
+  # It contains three key/value pairs:
+  #   awsAccessKeyIdName/awsAccessKeyIdVakue
+  #   awsSecretAccessKeyName/awsSecretAccessKeyValue
+  #   awsEndpointName/awsEndpointValue
   s3:
     bucket: ~
     secretName: ~
+    awsAccessKeyIdName: ~
+    awsSecretAccessKeyName: ~
+    awsEndpointName: ~
```

Comment:
We can probably do away with these, and enforce the secret keys to be the same as the environment variables.

```suggestion

```

Reply:
I think this way could be more configurable if the project already has the secret, but it is also true that it is bit confusing. let me think about it.

Reply:
I updated the key names in the secret to be same as the env variables